pkg/config/config.go

```diff
@@ -559,3 +558,35 @@ func concatenateBytes(data ...[]byte) []byte {
 	}
 	return bytes
 }
+
+// generateOrgConfigGroup returns an config group for a organization.
+// It defines the crypto material for the organization (its MSP).
+// It sets the mod_policy of all elements to "Admins".
+func generateOrgConfigGroup(org *Organization, mspConfig *mb.MSPConfig) (*cb.ConfigGroup, error) {
+	var err error
+
+	orgGroup := newConfigGroup()
+	orgGroup.ModPolicy = AdminsPolicyKey
+
+	if org.SkipAsForeign {
+		return orgGroup, nil
+	}
+
+	if err = addPolicies(orgGroup, org.Policies, AdminsPolicyKey); err != nil {
+		return nil, fmt.Errorf("failed to add policies: %v", err)
+	}
+
+	err = addValue(orgGroup, mspValue(mspConfig), AdminsPolicyKey)
+	if err != nil {
+		return nil, fmt.Errorf("failed to add msp value: %v", err)
+	}
+
+	if len(org.OrdererEndpoints) > 0 {
+		err = addValue(orgGroup, endpointsValue(org.OrdererEndpoints), AdminsPolicyKey)
+		if err != nil {
+			return nil, fmt.Errorf("failed to add orderer endpoints value: %v", err)
+		}
+	}
```

Comment:
This is specific to orderer orgs. It's fine doing that here since we share the same Organization type struct between all orgs (consortium, orderer, and application) but we should also consider using this path for application and consortium orgs. The consumer will pass the organization config determining whether orderer endpoints or anchor peers should be set so we shouldn't have to distinguish between what type of org we're creating a config group for.